src/test/config.ts

```diff
@@ -46,18 +37,18 @@ export const testAccounts = {
     email: 'superuser@test.com',
     password: 'Pa55word11',
     role: 'IDAM_SUPER_USER',
-    firstName: 'superTest'
+    firstName: 'superTest',
   },
   adminUser: {
     email: 'adminuser@test.com',
     password: 'Pa55word11',
     role: 'IDAM_ADMIN_USER',
-    firstName: 'adminTest'
+    firstName: 'adminTest',
```

Comment:
I wonder what is the benefit of adding a trailing comma here? In eslint.rc we set the lint rule to be "comma-dangle": ["error", "only-multiline"]...so it does not matter whether there is a trailing comma/

Reply:
Fixed